### DISCUSSION TITLE
如何在文章详情界面增加文章的宽度？我的布局是三栏

### DISCUSSION THREAD

**Discussion by W01fh4cker** [ppoffice/hexo-theme-icarus#1336] [Q&A]
![image](https://github.com/user-attachments/assets/5eb45439-63b7-45db-98e2-dd3702a6fe04)  
我的文章地址：https://w01fh4cker.github.io/2024/12/30/LearnJavaMemshellFromZero  
如题，两侧留白太多了，我希望减少留白，把中间的文章内容的宽度增加。

**Comment by AryCra07** [comment]
参考[这篇博客](https://blog.peiyingchi.com/2020/02/25/hexo-icarus-settings/#%E6%8A%8A%E6%96%87%E7%AB%A0%E7%95%8C%E9%9D%A2%E6%94%BE%E5%AE%BD)

**Reply by W01fh4cker** [reply]
> 参考[这篇博客](https://blog.peiyingchi.com/2020/02/25/hexo-icarus-settings/#%E6%8A%8A%E6%96%87%E7%AB%A0%E7%95%8C%E9%9D%A2%E6%94%BE%E5%AE%BD)

您好，我在https://github.com/ppoffice/hexo-theme-icarus/tree/master/source/css 下没有看到有_variables.styl文件呀

**Comment by forgetfulengineer** [comment]
我是新增了 [`source/css/forgetfulengineer.styl`](https://github.com/forgetfulengineer/hexo-theme-icarus/blob/master/source/css/forgetfulengineer.styl) 和 [`include/style/variable.styl`](https://github.com/forgetfulengineer/hexo-theme-icarus/blob/master/include/style/variable.styl) 來自定義主題配置，並且在 [`source/css/style.styl`](https://github.com/forgetfulengineer/hexo-theme-icarus/blob/master/source/css/style.styl) 載入設定

網站內容的寬度是由 [`node_modules\bulma-stylus\stylus\elements\container.styl`](https://github.com/groenroos/bulma-stylus/blob/master/stylus/elements/container.styl) 控制的，`$gap` 是左右的留白寬度，可以在 `include/style/variable.styl` 自定義留白寬度，並且增加更大螢幕尺寸的 `max-width` 來自適應大螢幕

> [詳細修改紀錄](https://github.com/forgetfulengineer/hexo-theme-icarus/commit/7e47bc55b0a83da8c6436bfc64391f9ab8d66cd0#diff-1a2ce4d4127d6b6d8b5cd247a1364dc0de7d2ada197dcad181e0dc3db9df9703R16)

歡迎參考我的[網站配置](https://github.com/forgetfulengineer/hexo-theme-icarus)，[網站](https://forgetfulengineer.github.io/)以 icarus 為基礎修改而成😁
